### DISCUSSION TITLE
Creating a Flux repo to kickstart GitOps working gp project

### DISCUSSION THREAD

**Discussion by monadic** [fluxcd/flux2#480] [Proposals] [answered]
Hi, as set out in [https://github.com/fluxcd/community/issues/39#issuecomment-730329517
](url)

The key points are: 
* a group of flux users/community who are in cncf want to create an oss project containing a document defining gitops "principles" (see issue linked above for more details)
* they all love fluxcd and its governance model, and have agreed to kickstart this in the cncf fluxcd org as "https://github.com/fluxcd/gitops-working-group"
* flux2 would be an example ref. impl. but the definition would also be usable by any impl. or team
* cncf sig app delivery is on board and will also create a working group with meetings etc.
* the agreed plan is to put this forward for cncf sandbox, keeping fluxcd governance.md, immediately, so project will move to its own org soonish

your understanding and support is sought!

alexis

**Comment by stefanprodan** [comment]
As a member of @fluxcd/oversight-committee I support this initiative 💯

**Comment by hiddeco** [comment]
+1 as a @fluxcd/oversight-committee member.

NB: link in post is broken, outline is defined in https://docs.google.com/document/d/11EZfvB2FFI837nMmArnyv-wizsIJvc-4_xdgfoUXF4o/view

**Comment by squaremo** [comment]
I think the idea of a GitOps working group as part of a CNCF SIG is a good one. I'm fine with giving it a home here until it can be a project in its own right. So, +1 from me for creating a repo `fluxcd/gitops-working-group`. Since that's all of @fluxcd/oversight-committee now in agreement, I think that can go ahead.

Two extra things:
 - it would be good to see people, especially those who would expect/like to be members of the working group, also comment with their support here
 - similarly, if people feel strongly that it is a bad idea for the repo to be in fluxcd/ (even given this is intended to be a temporary home), please set out your reasoning here too.

Thanks!

**Reply by dj1k** [reply]
The backers of the initiative have agreed to post their support on the readme entry in the new repo on the readme file once it's committed.

**Reply by squaremo** [reply]
Oh and another thing, you'll need to enumerate the initial maintainers.

**Comment by stefanprodan** [comment, marked as answer]
The gitops-working-group repository is here: https://github.com/fluxcd/gitops-working-group

**Reply by monadic** [reply]
oops - that's your personal copy!

**Comment by monadic** [comment]
The repo is here https://github.com/fluxcd/gitops-working-group
